### PR TITLE
Intel database

### LINKED ISSUE #27: selenium chrome driver update
the function chrome_browser_setup in scraping.py stopped working, probably due to an update in chromedriver url.
this seems to be a common issue (e.g., https://stackoverflow.com/questions/76727774/selenium-webdriver-chrome-115-stopped-working) but needs to be fixed in order for the program to work. 
some suggested solutions from stackoverflow are not working:
- updated webdriver_manager (4.0.0) and selenium (4.10.0)
- downloaded an older version of chromedriver (114) but is not compatible with local chrome version

### PR DESCRIPTION
**Pull request by giulianogiari**

### CODE REVIEW

File: src/utils/scraping.py [hunk 1]
```diff
@@ -1,74 +1,101 @@
...
+            # request the processor page
+            soup = BeautifulSoup(requests.get(processor_url).content, 'html.parser')
+            # stop the loop for a random time between 0 and 1 seconds to avoid being blocked
+            time.sleep(np.random.rand())
+            # get the tdp
+            processor_specs = soup.find_all("div", attrs={"class": "row tech-section-row"})
+            # iterate over specs
+            for spec in processor_specs:
+                # check if tech-label contains 'TDP'
+                if spec.find("div", attrs={"class": "tech-label"}) is not None:
```
Comment (MattiaSilvestri) [inline src/utils/scraping.py]: Isn't it more readible to just put `if spec.find("div", attrs={"class": "tech-label"}):` without the `is not None` part?

File: src/utils/scraping.py [hunk 2]
```diff
@@ -1,74 +1,101 @@
...
+                        tdp = spec.find("div", attrs={"class": "tech-data"}).text
+                        # retain only the numbers including the decimal points, if present
+                        tdp = float(re.findall(r'\d+\.\d+|\d+', tdp)[0])
+                        # store in dictionary
+                        intel_database[processor['data-value']] = tdp
+                        # break the loop over specs, to speed up the process
+                        break
+    # save the dictionary in a json file
+    # get the path to the data folder
+    path = [x[0] for x in os.walk('..') if 'data' in x[0]][0]
```
Comment (MattiaSilvestri) [inline src/utils/scraping.py]: Is there a reason why here you keep the old method of getting the path?

Comment (giulianogiari) [reply src/utils/scraping.py]: nope, indeed i should update also this line. thanks

**Review by FrnRmn — APPROVED** [review]
lgtm

**Review by MattiaSilvestri — APPROVED** [review]

**Pull request merged**